jaq-std/src/std.jq

```diff
@@ -22,6 +22,16 @@ def isinfinite: . == infinite or  . == -infinite;
 def isfinite:   isnumber and (isinfinite | not);
 def isnormal:   isnumber and ((. == 0 or isnan or isinfinite) | not);
 
+# Math
+def logb:
```

Comment:
I would create a test for this, for example using the inputs `(nan, infinite, -infinite, -2, -1, 0, 1, 2)`.

Reply:
Done!